source/derived/mus_derQuanPS_module.fpp

```diff
@@ -103,6 +103,7 @@ module mus_derQuanPS_module
 
   public :: mus_append_derVar_lbmPS
   public :: deriveEquilPS_FromMacro
+  public :: deriveEquilPS2_FromMacro
```

Comment:
Suggestion to rename routine `deriveEquilPS2ndOrder_FromMacro` to be self explaining.